recipe/meta.yaml

```diff
@@ -10,29 +10,33 @@ source:
   sha256: ee352333e8293e957ccb1b4597a111fc4f6da88ab74b8cb3f8f51eed1e12f500
 
 build:
-  noarch: python
   number: 0
-  script: {{ PYTHON }} -m pip install . --no-deps -vv
+  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vv
 
 requirements:
   host:
     - python
     - pip
+    - wheel
+    - setuptools
   run:
     - python
     - six
 
 test:
+  requires:
+    - pip
   imports:
     - flatten_json
+  commands:
+     - pip check
```

Comment:
Could you also add `flatten_json -- help` because of `entry_points`?

Reply:
@Arishamays1 Hm, something is wrong with `flatten_json --help`. Could you check it locally?

Reply:
Added, Thanks for the suggestion! It seems this package gets stuck on this test command when adding this to the meta.yaml both here onPrefect and locally. I will remove this as I do not believe it is a working feature.